### DISCUSSION TITLE
Installation d'EGSnrc sur Windows 10

### DISCUSSION THREAD

**Discussion by morad1991** [nrc-cnrc/EGSnrc#1068] [General]
![Capture](https://github.com/nrc-cnrc/EGSnrc/assets/90323580/a2cf6237-25d7-4881-9862-adbc184cea30)
Please help with this step

**Comment by rtownson** [comment]
If they are blank, this means that you haven't installed compilers or they are not in your PATH environment variable. Make sure to install mingw first, then try again.
